Tvl.VisualStudio.JustMyCodeToggle/JustMyCodeTogglePackage.cs

```diff
@@ -26,27 +29,19 @@ public JustMyCodeTogglePackage()
             _command = new OleMenuCommand(invokeHandler, changeHandler, beforeQueryStatus, id);
         }
 
-        public SVsServiceProvider ServiceProvider
-        {
-            get
-            {
-                return new VsServiceProviderWrapper(this);
-            }
-        }
-
         public EnvDTE.DTE ApplicationObject
         {
             get
             {
-                return ServiceProvider.GetService(typeof(EnvDTE._DTE)) as EnvDTE.DTE;
+                return GetService(typeof(EnvDTE._DTE)) as EnvDTE.DTE;
             }
         }
 
-        protected override void Initialize()
+        protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
         {
-            base.Initialize();
```

Comment:
:bulb: It appears code typically calls the base implementation, but it's not sure if this is required:

```csharp
await base.InitializeAsync(cancellationToken, progress);

// Rest of initialization ...
```

Reply:
I had checked that it was returning CompletedTask, but the very fact that I had to check makes me suspicious of looking in the first place. Funny enough, this was the last PR before I started leaving base calls in my overrides. :D